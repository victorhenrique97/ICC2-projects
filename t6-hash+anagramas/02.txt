cornelia
bedlar
alcor
acrolein
calor
creat
belard

