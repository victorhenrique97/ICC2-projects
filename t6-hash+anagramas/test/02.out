3
actinal, alantic, alicant, antical
amain, amani, amnia, anima
amentia, aminate, anamite, animate
